kubernetes-typescript/index.ts

```diff
@@ -0,0 +1,20 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as k8s from "@pulumi/kubernetes";
+
+const name = pulumi.getProject();
+
+const config = new pulumi.Config("kubeapp");
```

Comment:
This will be changed to `const config = new pulumi.Config();` and `kubeapp:image` in `Pulumi.yaml` will be changed to just `image` after https://github.com/pulumi/pulumi/issues/1768 lands and is released.

Reply:
I'm a little confused by this; shouldn't it be `const config = new pulumi.Config()`, so that it simply inherits the project name?

Reply:
Yes, it *should* be `const config = new pulumi.Config()`, but unfortunately it can't until https://github.com/pulumi/pulumi/issues/1768 is fixed.

Reply:
Oh, gotcha, so `new pulumi.Config()` effectively doesn't work as-is?